Ryujinx.Common/DisposableUnmanagedBase.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+
+namespace Ryujinx
+{
+    /// <summary>
+    /// A base class to implement IDisposable pattern for classes with unmanaged resources.
+    /// Use DisposableBase when there are only manage resources to dispose.
```

Comment:
```suggestion
    /// Use DisposableBase when there are only managed resources to dispose.
```
This also could be under "remarks".

Reply:
Good catch